api/src/controllers/contract.ts

```diff
@@ -53,7 +53,7 @@ export const accountTokenBalance = async (req: AppRequest<TokenBalanceParam>, re
       const token = await findERC20Token(req.body.contractAddress);
       res.send({ balance: 0, decimals: token.decimals });
     } else {
-      res.send({ balance: tokenBalances[0].balance, decimals: tokenBalances[0].decimals });
+      res.send({ balance: tokenBalances[0].balance, decimals: tokenBalances[0].info.decimals || 18 });
```

Comment:
why is decimals defaulted to 18?

Reply:
nfts do not have decimals, I've changed it to 0